CHANGELOG.rst

```diff
@@ -10,6 +10,8 @@ Changes since 2.0.1 include:
 - the new upgrade command, automatically executed on start
 - file statistics module and UI integration
 - file checksum verification
+- configuration options for B2NOTE integration
```

Comment:
- enable B2NOTE integration in production deployments via a configuration variable.